src/App.tsx

```diff
@@ -18,20 +18,26 @@ function App() {
       <Routes>
         <Route path="/" element={<Layout />}>
           <Route index element={<Home />}></Route>
-          <Route path="about" element={<About />}></Route>
+          <Route path="ask-doctor" element={<AskDoctor />}></Route>
           <Route path="auth">
-            <Route path="sign-up-doctor" element={
-              <QueryClientProvider client={queryClient}>
-                <SignUpDoctor />
-              </QueryClientProvider>
-            }></Route>
+            <Route
+              path="sign-up-doctor"
+              element={
+                <QueryClientProvider client={queryClient}>
```

Comment:
It's strange to wrap every route by QueryClientProvider, have already communicate with Kate that there are some issues with wrapping all App by QueryClientProvider, will try to fix them.

Reply:
Just tried on your setup, everything is working, could u please wrap the whole app with QueryClientProvider?
<QueryClientProvider client={queryClient}>
  <Routes>
    ....
  </Routes>
</QueryClientProvider>

Reply:
done